src/views/home/Review.vue

```diff
@@ -1,141 +1,167 @@
 <template>
   <div id="review">
-  <div class="container">
-    <img
-      class="book"
-      :src="content.avatar"
-    />
-    <div id="info">
-    <a><strong> {{ content.name }} </strong></a>
-    <br />
-    <a>Yazar: {{ content.owner }}</a>
-    <hr /> <br/>
-    <label for="review_rating">Puanım: <Rate @rate="rated"/></label> <br />
-    <div class="raf">
-      <form>
-        <label for="shelves">Raflar / durum: </label>
-        <WantedButton :contentType="content.type" @select="selectStatus"/>
-      </form>
-    </div>
-    <hr />
-    </div>
-    <label for="reviewselect">Ne düşünüyorsun ?</label>
-    <div class="right">
-      <div class="reviewselectlist">
-        <a style="color: #767676" href="#">Biçimlendirme ipuçları</a> |
-        <a style="color: #767676" href="#">Kitap/yazar ekle</a> |
-        <a style="color: #767676" href="#">Metin alanını büyüt</a>
+    <div class="container">
+      <img class="book" :src="content.image.path" />
+      <div id="info">
+        <a
+          ><strong> {{ content.name }} </strong></a
+        >
+        <br />
+        <a>Yazar: {{ content.user.name }}</a>
+        <hr />
+        <br />
+        <label for="review_rating">Puanım: <Rate @rate="rated" /></label> <br />
+        <div class="raf">
+          <form>
+            <label for="shelves">Raflar / durum: </label>
```

Comment:
Tüm yazılarda Türkçe değil lokalizasyon konusunda kolaylık için İngilizce kullanalım.